### PR TITLE
fix:  ADS 2.0 issue reported by QA & design team.

### PR DESCRIPTION
**Pull request by rohitagarwal88**
This PR fixes

1. Sentence case `Product update` in product update modal
2. Fixed multiple scroll issue. Product update modal use default modal scroll
3. Deduplicated Tag component.
4. Reduced section font-size to 16px
5. User profile page text converted to sentence case
6. `Display Upload Picture` is coming from old ads, and fix in the old ads.
7. Minor spacing alignment in user profile page
8. Auth page `Forget Password & Privacy Policy `converted  to sentence case
9. Header User profile has `cursor:pointer`
10. In Profile dropdown, `Edit Profile & Sign out` converted to sentence case
11. Text converted to sentence case, `New workspace, Admin settings, Join our discord, what's new, leave workspace, delete workspace, welcome tour`
12.  Updated blank workspace icon
Screenshot 2023-04-30 at 4 49 41 PM
=========

1. Updated Delete workspace icon
2. `Upload logo` is sentence case
3. Fixed Proceed button's spelling is incorrect in step 4 of guided tour
4. Removed position fixed from workspace settings
5. Alignment fixes in workspace setting to make it look consistent with Profile pages
6. Guide tour title color fix
7. [Tooltip is missing for Help](https://www.notion.so/appsmith/f353d8c6bd664f79ad858a42010cdfc8?v=f04cde23f6424aeb9d5a6e389cd172bd&p=1eb05a83510e40039f7aa1a40167c667&pm=s)
8. [Added google & github icon](https://www.notion.so/appsmith/f353d8c6bd664f79ad858a42010cdfc8?v=f04cde23f6424aeb9d5a6e389cd172bd&p=2c9f03ef12ec4a20ae26b92d53c5ef30&pm=s)

Fixes

1. [Templates issues](https://www.notion.so/appsmith/f353d8c6bd664f79ad858a42010cdfc8?v=f04cde23f6424aeb9d5a6e389cd172bd&p=2aa306cb8a10442683137705beba9e30&pm=s)
2. [Auth Pages & welcome screen](https://www.notion.so/appsmith/f353d8c6bd664f79ad858a42010cdfc8?v=f04cde23f6424aeb9d5a6e389cd172bd&p=9c3223fbdd43400ab352139e9dbe5674&pm=s)
3. [Workspace settings](https://www.notion.so/appsmith/f353d8c6bd664f79ad858a42010cdfc8?v=f04cde23f6424aeb9d5a6e389cd172bd&p=a36bc57f1bdd419d99b83135e22da1a3&pm=s)
4. [Templates Details Page](https://www.notion.so/appsmith/f353d8c6bd664f79ad858a42010cdfc8?v=f04cde23f6424aeb9d5a6e389cd172bd&p=6ee12af531df40febe03779cf87654a2&pm=s)
5. [Profile Page](https://www.notion.so/appsmith/f353d8c6bd664f79ad858a42010cdfc8?v=f04cde23f6424aeb9d5a6e389cd172bd&p=0819a22ecebc461f8bd1e791ad02e4f9&pm=s)
6. [Guided Tour](https://www.notion.so/appsmith/f353d8c6bd664f79ad858a42010cdfc8?v=f04cde23f6424aeb9d5a6e389cd172bd&p=ad94d21f04ae4a8f944bae05e35c92ac&pm=s)
7. [ Menu opens and closed with a 2-3 sec delay.](https://www.notion.so/appsmith/f353d8c6bd664f79ad858a42010cdfc8?v=f04cde23f6424aeb9d5a6e389cd172bd&p=4932b83c70354201a6eea32891628df2&pm=s)
8. [Application page issues](https://www.notion.so/appsmith/f353d8c6bd664f79ad858a42010cdfc8?v=f04cde23f6424aeb9d5a6e389cd172bd&p=7da9a5f1cc01435fb7bac132f9516b00&pm=s)
9. [Menu isn’t getting dismissed which is causing a break in UI flow for actions](https://www.notion.so/appsmith/f353d8c6bd664f79ad858a42010cdfc8?v=f04cde23f6424aeb9d5a6e389cd172bd&p=bbf9d13bd5f04c528cced282b7732506&pm=s)
10. [Hover state should not be present for the user details since it isn’t a clickable element](https://www.notion.so/appsmith/f353d8c6bd664f79ad858a42010cdfc8?v=f04cde23f6424aeb9d5a6e389cd172bd&p=254669ad47494695afb6a6bde02c847b&pm=s)
11. [Preview is disabled](https://www.notion.so/appsmith/f353d8c6bd664f79ad858a42010cdfc8?v=f04cde23f6424aeb9d5a6e389cd172bd&p=dc1682b981034e20bf25045010758a72&pm=s)
12. [Arrow for close side bar should turn towards the left](https://www.notion.so/appsmith/f353d8c6bd664f79ad858a42010cdfc8?v=f04cde23f6424aeb9d5a6e389cd172bd&p=f75f41ca1fa2499db01984cb1b247db4&pm=s)
13. [Arrow for lock sidebar open should turn towards the right](https://www.notion.so/appsmith/f353d8c6bd664f79ad858a42010cdfc8?v=f04cde23f6424aeb9d5a6e389cd172bd&p=a3c3180e82034ef28f8a0e75c245bc38&pm=s)
14. [Unable to scroll to the bottom of the list](https://www.notion.so/appsmith/f353d8c6bd664f79ad858a42010cdfc8?v=f04cde23f6424aeb9d5a6e389cd172bd&p=bb29c34c797d4c0f9eb67db2f5f478c1&pm=s)
15. [Header - Clarifications](https://www.notion.so/appsmith/f353d8c6bd664f79ad858a42010cdfc8?v=f04cde23f6424aeb9d5a6e389cd172bd&p=6af2854856d34578a98b8a85089e6950&pm=s)
16. [Onboarding issues](https://www.notion.so/appsmith/f353d8c6bd664f79ad858a42010cdfc8?v=f04cde23f6424aeb9d5a6e389cd172bd&p=18b7caaa8da240f6b29570055e9ecfd8&pm=s)
17. [Wrong icon for edit in applications page](https://www.notion.so/appsmith/f353d8c6bd664f79ad858a42010cdfc8?v=f04cde23f6424aeb9d5a6e389cd172bd&p=54c03f0e15fd4cec944e723c83f3e2fd&pm=s)
18. [Tooltip missing over help](https://www.notion.so/appsmith/f353d8c6bd664f79ad858a42010cdfc8?v=f04cde23f6424aeb9d5a6e389cd172bd&p=1eb05a83510e40039f7aa1a40167c667&pm=s)
19. [Go-to icon to be seen on hover ](https://www.notion.so/appsmith/f353d8c6bd664f79ad858a42010cdfc8?v=f04cde23f6424aeb9d5a6e389cd172bd&p=680fa5fac5124224aec8cea56baa89a7&pm=s)

## Type of change

> Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Chore (housekeeping or task changes that don't impact user perception)
- This change requires a documentation update

- Manual
- Jest
- Cypress

### Dev activity
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

**Review by albinAppsmith — APPROVED** [review]

File: app/client/src/pages/workspace/settings.tsx [hunk 1]
```diff
@@ -46,31 +46,29 @@ const SentryRoute = Sentry.withSentryRouting(Route);
...
     height: 100%;
-    ${({ isMobile }) =>
-      !isMobile &&
-      `
-      padding: 110px 0 0;
-  `}
   }
 `;
 
-const StyledStickyHeader = styled(StickyHeader)<{ isMobile?: boolean }>`
```
Comment (ankitakinger) [inline app/client/src/pages/workspace/settings.tsx:65]: Why exactly are we removing `StickyHeader` here? This was intentionally implemented earlier. @rohitagarwal88

Comment (rohitagarwal88) [reply app/client/src/pages/workspace/settings.tsx:65]: @ankitakinger : I'll bring it back. thank for reviewing it.

**Comment by laveena-en** [comment]
@akash-codemonk Profile Page: White block behind the toast is still seen behind the toast. Please try with the below image file: 
image
image

A grey BG colour is seen behind the remove button which isn’t present on release. Also, the text colour seems incorrect. @vasanthappsmith Can you please confirm?

Users list layout is broken
image

Primary button to be used in the modal 
image

Applications page:
Performance delay in menu button still exists

cursor pointer isn’t seen when a dropdown is open https://www.loom.com/share/366cc6515bbb4736b7015f97850e617d

**Comment by rohitagarwal88** [comment]
Users list layout is broken --> **Fixed after merging epic**
image

Primary button to be used in the modal -> @laveena-en : `all delete button will be error button`

**Pull request merged**